plone/app/linkintegrity/tests/test_handlers.py

```diff
@@ -0,0 +1,35 @@
+from plone.app.linkintegrity.tests.base import ATBaseTestCase
+from plone.app.linkintegrity.tests.base import DXBaseTestCase
+from plone.app.linkintegrity.handlers import findObject
+
+
+class FindObjectTests:
```

Comment:
In Python 2, not inheriting from object will create an old-style class (different MRO), please inherit from object to avoid this.